scripts/zkai_d64_nested_verifier_backend_spike_gate.py

```
@@ -0,0 +1,748 @@
+        mutator(mutated)
+        cases.append((name, surface, mutated))
+
+    add("safe_checkpoint_commit_drift", "safe_checkpoint", lambda p: p["safe_checkpoint"].__setitem__("main_commit", "0" * 40))
+    add("source_contract_path_drift", "source_contract_evidence", lambda p: p["source_contract_evidence"].__setitem__("path", "docs/engineering/evidence/../evidence/zkai-d64-nested-verifier-backend-contract-2026-05.json"))
+    add("source_contract_file_hash_drift", "source_contract_evidence", lambda p: p["source_contract_evidence"].__setitem__("file_sha256", "11" * 32))
+    add("source_contract_payload_hash_drift", "source_contract_evidence", lambda p: p["source_contract_evidence"].__setitem__("payload_sha256", "22" * 32))
+    add("source_contract_result_drift", "source_contract_evidence", lambda p: p["source_contract_evidence"].__setitem__("contract_result", "NO_GO"))
+    add("nested_verifier_contract_commitment_drift", "nested_verifier_contract", lambda p: p.__setitem__("nested_verifier_contract_commitment", "blake2b-256:" + "33" * 32))
+    add("selected_slice_id_drift", "nested_verifier_contract", lambda p: p["selected_slice_ids"].__setitem__(1, "tampered_bridge"))
+    add("candidate_inventory_status_relabel", "candidate_inventory", lambda p: p["backend_attempt"]["candidate_inventory"][2].__setitem__("status", "OUTER_PROOF_VERIFIED"))
+    add("candidate_inventory_acceptance_relabel", "candidate_inventory", lambda p: p["backend_attempt"]["candidate_inventory"][2].__setitem__("accepted_as_outer_backend", True))
```

Ruff (0.15.12)

[warning] Boolean positional value in function call

(FBT003)